src/contracts/middleware/BLSPubkeyRegistry.sol

```diff
@@ -91,6 +91,22 @@ contract BLSPubkeyRegistry is IBLSPubkeyRegistry, Test {
         return pubkeyHash;
     }
 
+    /// @notice Returns the index of the quorumApk index at `blockNumber` for the provided `quorumNumber`
+    function getApkIndicesForQuorumsAtBlockNumber(bytes calldata quourmNumbers, uint256 blockNumber) external view returns(uint32[] memory){
+        uint256[] memory indices = new uint256[](quourmNumbers.length);
+        for (uint i = 0; i < quourmNumbers.length; i++) {
+            uint8 quorumNumber = uint8(quourmNumbers[i]);
+            uint32 length = uint32(quorumApkUpdates[quorumNumber].length);
```

Comment:
I feel like this should be named something like `qorumApkUpdatesLength` instead of just `length`

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/81/commits/d4daf8507a5799729702562ebdd1e37b6f88caee